### PR TITLE
Remove flyer email a/b test

### PR DESCRIPTION
**Pull request by davidfurey**
## What does this change?

Removes flyer email a/b test. Also switches the default list id to 3806, requested by Celine.  Follow up PR will remove the aliases once the list ids have been merged.

## What is the value of this and can you measure success?

Test is complete, this is just tidying up

## Does this affect other platforms - Amp, Apps, etc?

No

## Tested in CODE?

No

### CODE REVIEW

File: common/app/model/EmailNewsletters.scala [hunk 1]
```diff
@@ -296,9 +296,9 @@ object EmailNewsletters {
     teaser = "Uncover unconventional destinations and rediscover old favourites - let our travel editors guide you to trips worth taking",
     description = "Weekly travel inspiration. Off-piste attractions, budget breaks, top 10s and reader reviews. Uncover unconventional destinations and rediscover old favourites - let our travel editors guide you to trips worth taking",
     frequency = "Every Wednesday",
-    listId = 2211,
+    listId = 3806,
     tone = Some("feature"),
-    aliases = List(3806, 3807),
+    aliases = List(2211, 3807),
     signupPage = Some("/travel/2016/aug/18/sign-up-to-the-flyer"),
     exampleUrl = Some("/email/the-flyer?format=email-connected")
```
Comment (katebee) [inline common/app/model/EmailNewsletters.scala]: `connected` was associated with 3807? The exampleUrl should be switched to the `cards` (?3806) version:
https://www.theguardian.com/email/the-flyer?format=email

Comment (davidfurey) [inline common/app/model/EmailNewsletters.scala]: All three lists are currently receiving the same email, so it shouldn't matter.  Celine asked for users to be moved to 3806.

Comment (katebee) [inline common/app/model/EmailNewsletters.scala]: So in `newsletters.scala`, the exampleUrl was set to a default, and then [there was a function](https://github.com/guardian/frontend/pull/16562/files#diff-9d741a60bc0d1397c123deea81ec79bcL37) in the ab test that set the exampleUrl to the correct preview according to the user test bucket.

The ab test is being deleted, so there is nothing to change this now... so everyone will see the  `?format=email-connected` preview. Not sure how much of an issue it is? (minor?) 😁

**Comment by davidfurey** [comment]
(🚢 from Kate)

**Pull request merged**